### ISSUE TITLE
Key manager with webpack plugin question

### ISSUE THREAD

**Issue by shoffman-applause** [ngneat/transloco-keys-manager#5]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Performance issue
[ ] Feature request
[x] Documentation issue or request
[ ] Support request
[ ] Other... Please describe:
</code></pre>

## Current behavior

I'm trying to use key manager to manage keys within my project with the webpack plugin. I have an english translation file (en.json) as my 'master' translation file and several other sample ones that aren't in sync with the master copy(the master copy would be periodically translated into various languages and merged back into my repo).

I want the plugin to manage the english translation file ONLY and not the other ones. Reason for this is that I have transloco configured to fallback to english if a specific translation key doesn't exist. When key manager manages other files it puts in the default value for the key which I don't want because I'll lose the english fallback. I have this configured in my `transloco.config.js` file and it works the first time I run `ng serve`. However, if I make changes to the project while the `ng serve` command is still running it then ignores this configuration and changes all the existing translation files.

Is there a way to make my use case work?

Here is my `transloco.config.js` file config:
```
module.exports = {
  rootTranslationsPath: 'src/assets/i18n/',
  langs: ['en'],
  keysManager: {
    defaultValue: 'Key Missing'
  },
  unflat: true,
  marker: 'dynamic-'
};
```

And here is my webpack-dev.config.js:
```
const { TranslocoExtractKeysWebpackPlugin } = require('@ngneat/transloco-keys-manager');
const translocoConfig  = require('./transloco.config.js')
module.exports = {
  plugins: [new TranslocoExtractKeysWebpackPlugin(translocoConfig)]
};
```
## Expected behavior

That the key manager webpack plugin would respect the configuration in `transloco.config.js` even during code changes.

## Environment
@angular-devkit/architect         0.803.19
@angular-devkit/build-angular     0.803.19
@angular-devkit/build-optimizer   0.803.19
@angular-devkit/build-webpack     0.803.19
@angular-devkit/core              8.3.19
@angular-devkit/schematics        8.3.19
@angular/cdk                      8.2.3
@angular/cli                      8.3.19
@angular/material                 8.2.3
@ngtools/webpack                  8.3.19
@schematics/angular               8.3.19
@schematics/update                0.803.19
rxjs                              6.5.3
typescript                        3.5.3
webpack                           4.39.2

 
For Tooling issues:
- Node version: 12.13.0
- Platform:  Mac

**Comment by NetanelBasal** [comment]
> However, if I make changes to the project while the ng serve command is still running it then ignores this configuration **and changes all the existing translation files.**

Can you explain what it means, please?

**Comment by shoffman-applause** [comment]
The first time the server starts up during `ng serve`, the webpack plugin uses the config I pasted above correctly and only updates the english translation file. If I make any other change to the codebase while the angular dev server is running (that would trigger a hot reload), the plugin will ignore the configuration and update every translation file, not just the english one.

Does that help?

**Comment by shaharkazaz** [comment]
@shoffman-applause Thank you for your report, I'll fix this issue 🙂

**Comment by shaharkazaz** [comment]
@shoffman-applause This will be available in v1.2.0 that will be released tomorrow morning 🙂
